shared_model/interfaces/permissions.hpp

```diff
@@ -98,9 +99,13 @@ namespace shared_model {
      public:
       PermissionSet();
       PermissionSet(std::initializer_list<Perm> list);
+      PermissionSet(const std::string &s);
 
-      size_t size() const;
+      std::string to_string() const;
```

Comment:
Naming does not follow our codestyle. Also maybe specify that it is a bitstring?

Again maybe the wrong abstraction here